hugegraph-server/hugegraph-core/src/main/java/org/apache/hugegraph/backend/serializer/BinaryBackendEntry.java

```diff
@@ -207,10 +208,10 @@ public int hashCode() {
         return this.id().hashCode() ^ this.columns.size();
     }
 
-    public static final class BinaryId implements Id {
+    public static class BinaryId implements Id {
```

Comment:
seems don't need to transform BinaryId, it's just short-term used during serialization. you cam double check

Reply:
ok